packages-web/fieldset/webpack.config.prod.js

```diff
@@ -1 +0,0 @@
-module.exports = require("../../configs/webpack.prod.js");
```

Comment:
Does this mean that you removed all usages of these webpack configs? Can we then remove them?

Reply:
For PIW yes, removed from all :)